spec/requests/user_request_spec.rb

```diff
@@ -1,13 +1,14 @@
 require 'rails_helper'
 
-RSpec.describe 'User controller' do
-  it 'should return a user' do
-    get '/users/1'
-    expect(response).to have_http_status(:success)
-    expect(response.status).to eq(200)
-    expect(response.body).to include('name')
-    expect(response.body).to include('id')
-    expect(response.body).to include('Number of posts:')
-    expect(response.body).to include('image')
+describe 'Testing the user controller' do
+  it 'renders a successful response' do
+    user = User.create!(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
+                        posts_counter: 0)
+    get user_url(user)
+    expect(response).to be_successful
+    expect(response).to have_http_status(200)
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=to%20your%20project.-,Create%20a%20Request%20spec%20file%20for%20all%20your%20controllers.,-Make%20sure%20that), You need to test both index and show for each controller, So Kindly add a request tests to the `index `path as you did with `show `path. and also make sure that the test type is `request`.
[Hint] It will be like that:
```ruby
require 'rails_helper'

RSpec.describe 'Users', type: :request do
  describe 'GET /index' do
    before(:example) { get '/users' }
    it 'the request is a success' do
      expect(response).to have_http_status(:ok)
    end
    it 'Renders the right template' do
      expect(response).to render_template('index')
    end
    it 'Include the correct text' do
      expect(response.body).to include('whatever text in the page')
    end
  end

  describe 'GET /show' do
    before(:example) { get '/users/:id' }
    it 'the request is a success' do
      expect(response).to have_http_status(:ok)
    end
    it 'Renders the right template' do
      expect(response).to render_template('show')
    end
    it 'Include the correct text' do
      expect(response.body).to include('whatever text in the page')
    end
  end
end
```